components/nimbus/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "nimbus-sdk"
+version = "0.10.0"
+authors = ["The Glean Team <glean-team@mozilla.com>", "The Sync Team <sync-team@mozilla.com>"]
+edition = "2018"
+description = "A rapid experiment library"
+readme = "README.md"
+license = "MPL-2.0"
+keywords = ["experiment", "nimbus"]
+
+[lib]
+name = "nimbus"
+crate-type = ["lib", "cdylib"]
+
+[features]
+# `rkv-safe-mode` is the same name as used by glean. For us though, it's the default.
+default=["rkv-safe-mode", "uniffi-bindings"]
+rkv-safe-mode = []
+uniffi-bindings = ["uniffi", "uniffi_build"]
+
+[dependencies]
+anyhow = "1"
+serde = "1"
+serde_derive = "1"
+serde_json = "1"
+log = "0.4"
+# Viaduct locked to version same version of app-services as mozilla-central
+viaduct = { path = "../viaduct" }
+thiserror = "1"
+url = "2.1"
+rkv = "0.17"
+jexl-eval = "0.1.7"
+uuid = { version = "0.8", features = ["serde", "v4"]}
+sha2 = "0.9"
+hex = "0.4"
+once_cell = "1"
+uniffi = { version = "^0.8.0", optional = true }
+
+[build-dependencies]
+uniffi_build = { version = "^0.8.0", features = [ "builtin-bindgen" ], optional = true }
+
+[dev-dependencies]
+viaduct-reqwest = { path = "../support/viaduct-reqwest" }
+mockito = "0.27"
+env_logger = "0.7"
+clap = "2.33.3"
+tempdir = "0.3"
+
+[package.metadata.release]
```

Comment:
nit: we should remove this `cargo release` config, it will no longer be used because we'll no longer be using `cargo release`.

Reply:
Fixed.